app/controllers/api/movies_controller.rb

```diff
@@ -1,24 +1,31 @@
 class Api::MoviesController < ApplicationController
-  before_action :authenticate_user, only: :user_tier_render
-
-  def index    
+  def index
     netflix = Rails.application.credentials.unogsng_key
 
     begin
-      response = RestClient.get('https://unogsng.p.rapidapi.com/search?type=movie&orderby=rating',
-                                headers = { 'x-rapidapi-key': netflix })
-      higher_rated_movies = netflix_sort(response)
-      
-      if params.has_key?(:lat) && params.has_key?(:lon)
```

Comment:
Why you movie this down? its the same thing right?

Reply:
Yes it is the same thing. We don't want to get rid of this functionality, we wrapped it with another if statement that checks if the user is making a query search for title.

Reply:
Allright!